open-class/docs/classes/assignment-branch-logic.md

```diff
@@ -10,244 +10,187 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 ## Walking Down the Path: Branching Logic
 
 <img alt="Woman at crossroads" src={useBaseUrl('img/einar-storsul-K3W7I7x37Xk-unsplash.jpg')} />
+
 _Photo by Einar Storsul on Unsplash_
 
-All but the most complex Docassemble interviews will have some choices that change the information
-that you show to the user or that you will need to ask a follow-up question for. Sometimes, this choice
-making is called branching logic. It's the kind of choice that you might think of making with a flowchart.
+Very simple interviews might just ask a user some questions and spit out some result. Most interviews, though, will want to change what the questions they ask and the results they show based on a user's answers.
+
+Our code has to make a choice and, until the AI take over, we have to give it very specific instructions about how to make that choice. That's often called *branching logic*. Probably because of trees.
 
-In today's class, we'll talk about how to write a Docassemble interview that achieves this goal.
+## Example 1
 
-## The problem
+Suppose you have a friend who comes to you for advice. They were invited to a party, but they're not sure if they should go. You give them advice, your friend is happy, and now you have lots of friends, and even strangers coming to you looking for advice on the same topic. You decide [code is the answer](https://xkcd.com/1319/) and you'll write a simple form to help them out.
 
-Suppose you have a friend who comes to you for advice. They were invited to a party, but they're not
-sure if they should go. You give them advice, your friend is happy, and now you have lots of friends coming
-to you looking for advice on the same topic. You decide to write a simple flowchart to help them out.
+You start simple.
 
 ### Thinking through outcomes
 
-What outcomes will our flowchart have?
+What outcomes will our flowchart have? 
 
 1. Go to the party
 2. Stay home
-3. Go, but don't stay too late
-
-### Gathering the facts
-
-What kind of information do we want to know to advise our friends? Let's think of some possible questions.
-
-* Do you like parties?
-* Is there anyone there who you want to meet?
-* How difficult is it to get to the party?
-* Do you have any big assignments or studying you should do?
```

Comment:
Note: For me, when I start in on a new topic, I try to focus on one concept at a time and keep it as simple as I can think of. I often find it seductive to add more content, but I try hard to ask if it's absolutely necessary for covering the one thing I'm focusing on at the moment.